src/main/resources/reference.conf

```diff
@@ -1674,6 +1677,35 @@ resourceTypes = {
     allowLeaving = false
     reuseIds = true
   }
+
+  cbas-submission = {
+    actionPatterns = {
+      abort = {
+        description = "Abort this cbas-submission"
+      }
+      read = {
+        description = "read information about the submission"
+      }
+      create_with_parent = {
+        description = "Enables creating the request object with a parent"
+      }
+      read_creator = {
+        description = "Enables reading the creator of the resource"
+      }
+    }
+    ownerRoleName = "owner"
+    roles = {
+      owner = {
+        roleActions = ["abort", "create_with_parent"]
+        includedRoles = ["reader"]
+      }
```

Comment:
How hard is it to change these after creation if we wanted to (say) have a "canceller" role separate from the owner/creator?